templates/package.json.template

```diff
@@ -7,14 +7,20 @@
 	"private": true,
 	"dependencies": {},
 	"devDependencies": {
-		"@wordpress/browserslist-config": "^4.1.0",
-		"@wordpress/env": "^4.1.0",
-		"@wordpress/scripts": "^17.0.0"
+		"@wordpress/browserslist-config": "^4.1.2",
+		"@wordpress/env": "^4.6.0",
+		"@wordpress/scripts": "^22.5.0"
 	},
 	"scripts": {
+		"initial-setup": "yarn && composer install && yarn run update-configs",
+		"update-configs": "TEXTDOMAIN=wporg composer exec update-configs",
```

Comment:
I ended up renaming these to `setup:tools` & `update:tools` in the new parent theme project, what do you think about adopting that here?

I also [have `setup:wp`](https://github.com/WordPress/wporg-parent-2021/blob/588837d8d7654603f026b1d3dccd4915cf397bfb/package.json#L14) which runs the setup shell scripts & imports.

Reply:
👍 Those look good to me. Feel free to add them to this PR.